### PR TITLE
MC-13777: added initial recollect docs

### PR DESCRIPTION
**Pull request by jasondias9**
#### Changes made
Updated the existing rollback api docs to show support for the `RECOLLECT` type + add more description
#### Related PRs
- [Core](https://github.com/pulls?q=is%3Apr+is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc)

### CODE REVIEW

File: source/includes/administration/_rollbacks.md [hunk 1]
```diff
@@ -153,17 +157,25 @@ curl "https://cloudmc_endpoint/rest/rollbacks/23910576-d29f-4c14-b663-31d728ff49
...
+Creates a rollback which will result in reprocessing of usage for the given request body. The Rollback API supports two types of rollback. `REPROCESS` and `RECOLLECT`
 
-The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin. 
-
-> Note: You must have the Reseller pricing permission on the target reseller. If the caller does not have the correct permissions or the reseller with the ID provided does not exist then a `404 Not Found` response will be returned.
+The `REPROCESS` rollback type uses existing usage that was previously collected from a backend service and re-processes it in the monetization engine. The latest pricing configuration will be used when these records. 
+The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin. If usage is not found for for the requested date and organization and service connection pair the API will default to the earliest available usage for the pair. If there is no previous usage for the organization and connection, no rollback will be generated for this pair.
```
Comment (razine-bensari) [inline source/includes/administration/_rollbacks.md]:
```suggestion
The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin the reprocess. If usage is not found for the requested date and organization and service connection pair the API will default to the earliest available usage for the pair. If there is no previous usage for the organization and connection, no rollback will be generated for this pair.
```

File: source/includes/administration/_rollbacks.md [hunk 2]
```diff
@@ -153,17 +157,25 @@ curl "https://cloudmc_endpoint/rest/rollbacks/23910576-d29f-4c14-b663-31d728ff49
...
 
 <aside class="notice">
   <strong>Note:</strong> A rollback will be performed for every valid pair of organizations and service connections. Rollbacks will only be performed when the organization is assigned the connection and has some previous usage for this connection.
 </aside>
 
+The `RECOLLECT` rollback type can be used by connection owners to retrigger the collection of usage from the underlying backend service. The `RECOLLECT` rollback is limitted by the retention period of underlying usage of the targeted service. If for example the targeted service only stores 60 days of usage the earliest reset date will be 60 days before today. A recollect rollback impacts **all** assigned organizations of the selected service connection.
```
Comment (razine-bensari) [inline source/includes/administration/_rollbacks.md]:
```suggestion
The `RECOLLECT` rollback type can be used by connection owners to retrigger the collection of usage from the underlying backend service. The `RECOLLECT` rollback is limitted by the retention period of underlying usage of the targeted service. If, for example, the targeted service only stores 60 days of usage the earliest reset date will be 60 days before today. A recollect rollback impacts **all** assigned organizations of the selected service connection. 
```

**Review by razine-bensari — APPROVED** [review]
Added a comment to fix typo

File: source/includes/administration/_rollbacks.md (hunk 1 above)
Comment (razine-bensari) [reply source/includes/administration/_rollbacks.md]: feel free to disregard the comment about the commas if it does not make sense lol.

File: source/includes/administration/_rollbacks.md (hunk 2 above)
Comment (razine-bensari) [reply source/includes/administration/_rollbacks.md]: I think we need commas but not sure

Comment (jasondias9) [reply source/includes/administration/_rollbacks.md]: I think you are right

File: source/includes/administration/_rollbacks.md [hunk 3]
```diff
@@ -153,17 +157,25 @@ curl "https://cloudmc_endpoint/rest/rollbacks/23910576-d29f-4c14-b663-31d728ff49
...
 `POST /rollbacks`
 
-Creates a rollback which will result in reprocessing of usage for the given request body. 
-This API currently supports the REPROCESS rollback type which uses existing usage previously collected by plugin usage collectors and re processes it in the monetization engine. The latest pricing configuration will be used when processing the records. 
+Creates a rollback which will result in reprocessing of usage for the given request body. The Rollback API supports two types of rollback. `REPROCESS` and `RECOLLECT`
 
-The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin. 
-
-> Note: You must have the Reseller pricing permission on the target reseller. If the caller does not have the correct permissions or the reseller with the ID provided does not exist then a `404 Not Found` response will be returned.
+The `REPROCESS` rollback type uses existing usage that was previously collected from a backend service and re-processes it in the monetization engine. The latest pricing configuration will be used when these records.
```
Comment (m-orfanos) [inline source/includes/administration/_rollbacks.md]: "uses existing usage" sounds a bit off imo. There's also a typo in the 2nd sentence right?

```suggestion
The `REPROCESS` rollback re-processes existing usage, previously collected from the backend service, in the monetization engine. The latest pricing configuration will be used on these records. 
```

**Review by m-orfanos — APPROVED** [review]

File: source/includes/administration/_rollbacks.md (hunk 3 above)
Comment (jasondias9) [reply source/includes/administration/_rollbacks.md]: Thanks... Sorry I missed that.

**Pull request merged**